src/pr-allowed.js

```diff
@@ -15,6 +15,44 @@ async function prAllowed(options) {
   }
 }
 
+async function jobsErrorEnabled(options) {
+  switch (await getErrorPolicy(options)) {
+    case false:
+      return false;
+
+    case true:
+      return true;
+
+    default:
+      return false;
+  }
+}
+
+async function getErrorPolicy({login, organization, repository, instGithub, debug}) {
```

Comment:
I suggest adding another  parameter to `getRepoPolicy` to check for the appropriate field(`allowPullRequests` or `enableJobsError`), rather than creating another function which does almost the same thing. 
Let's wait for Dustin's comments though :)

Reply:
Thank you @ydidwania  :) That'd help reduce the redundancy.